features/filesystem/bd/MBRBlockDevice.cpp

```diff
@@ -96,6 +96,7 @@ static int partition_absolute(
     table->entries[part-1].type = type;
 
     // lba dimensions
+    MBED_ASSERT(bd->is_valid_erase(offset, size));
```

Comment:
Ah, lba_offset and lba_size is in units of erase blocks. The offset and size are in units of bytes. bd->is_valid_erase operates in units of bytes.

We could also have an assert on lba_offset and lba_size but I thought it would be redundant. Do you think we should have that as well?

Reply:
No, don't bother.